lib/compat/wordpress-6.5/html-api/class-gutenberg-html-tag-processor-6-5.php

```diff
@@ -298,8 +298,8 @@
  *
  * The special elements are:
  *  - `SCRIPT` whose contents are treated as raw plaintext but supports a legacy
- *    style of including Javascript inside of HTML comments to avoid accidentally
- *    closing the SCRIPT from inside a Javascript string. E.g. `console.log( '</script>' )`.
+ *    style of including JavaScript inside of HTML comments to avoid accidentally
+ *    closing the SCRIPT from inside a JavaScript string. E.g. `console.log( '</script>' )`.
```

Comment:
This is backported code, and will disappear once WordPress 6.7 is shipped. It may not be worth introducing the divergence from Core and these "built assets." In other words, this is akin to modifying the contents of dependencies inside `node_modules`. Just wanted to point this out, as it has and can cause confusion when the code is different than in Core for the given version, beyond the `Gutenberg_` prefix naming.

Reply:
@dmsnell I never know which order I should update compat code. I've pushed the equivalent branch to WP-Dev with the goal of getting it in to WP 6.6 to update the developer docs on wp.org. See https://github.com/WordPress/wordpress-develop/pull/6785